wdn/templates_4.1/scripts/script-font.js

```diff
@@ -0,0 +1,25 @@
+define([
+	'jquery',
+	'css!js-css/script-font'
+], function($, WDN, require) {
+    var initd = false;
+
+    return {
+        initialize: function() {
+            // protect against multiple initializations
+            if (initd) {
+                return;
+            }
+            initd = true;
+
+            // Load Sanelma script font
+            var $scriptFont = $("<link>", {
+                "rel" : "stylesheet",
+                "type" : "text/css",
+                "href" : "js-css/script-font.css"
```

Comment:
I'm confused, this CSS file is already in the require statement. Why are you adding it again with this link element.?